model/src/main/java/de/danoeh/antennapod/model/feed/FeedPreferences.java

```diff
@@ -60,7 +60,7 @@ public static NewEpisodesAction fromCode(int code) {
     }
 
     public enum SkipSilence {
-        OFF(0), GLOBAL(1), AGGRESSIVE(2);
+        GLOBAL(0), OFF(1), MEDIUM(2), AGGRESSIVE(3);
```

Comment:
Please keep the IDs as they are. They don't have to be sorted but the meaning should stay the same. Otherwise this changes the existing settings for users when they upgrade.